bootstrap.sh

```diff
@@ -1 +1,21 @@
 #!/bin/bash
+
+kubectl apply -f .infrastructure/namespace.yml
```

Comment:
Ensure that the file path `.infrastructure/namespace.yml` is correct and matches your directory structure.